sdk/resourcehealth/Azure.ResourceManager.ResourceHealth/api/Azure.ResourceManager.ResourceHealth.netstandard2.0.cs

```diff
@@ -1,56 +1,49 @@
 namespace Azure.ResourceManager.ResourceHealth
 {
-    public partial class AvailabilityStatusData : Azure.ResourceManager.Models.ResourceData
+    public partial class ResourceHealthAvailabilityStatusData : Azure.ResourceManager.Models.ResourceData
     {
-        internal AvailabilityStatusData() { }
+        internal ResourceHealthAvailabilityStatusData() { }
         public Azure.Core.AzureLocation? Location { get { throw null; } }
-        public Azure.ResourceManager.ResourceHealth.Models.AvailabilityStatusProperties Properties { get { throw null; } }
+        public Azure.ResourceManager.ResourceHealth.Models.ResourceHealthAvailabilityStatusProperties Properties { get { throw null; } }
     }
-    public partial class AvailabilityStatusResource : Azure.ResourceManager.ArmResource
+    public partial class ResourceHealthAvailabilityStatusResource : Azure.ResourceManager.ArmResource
```

Comment:
I personally think this and ResourceHealthChildAvailabilityStatusResource do not make much sense as resources. Please consider making these non-resources.
Its name suggests it is fetch some kind of status, this does not look like a resource

Reply:
fixed